### PR TITLE
AST-1598 - Build: Release PR for v3.7.8

### PR DESCRIPTION
**Pull request by nileshc-bsf**
### Description
- Final release PR. Please check all the checklist below before merging.
---

### Checklist:
- [x] Finalize changelog.txt
- [x] Add doc links in the changelog.txt
- [x] Change minimum addon update version - ASTRA_EXT_MIN_VER
- [x] Run to update packages        : `npm audit fix`
- [x] Run to produce minified files : `grunt minify`
- [x] Run to update Google Fonts    : `grunt google-fonts`
- [x] PHPCS - vendor/bin/phpcs
- [x] Run to update version number  : `grunt version-bump --ver=<version-number>`
- [x] Check the Background Process function version
- [x] Update - Tested Upto - style.css & readme.txt
- [x] Update Readme.txt file and then run: `grunt readme`
- [x] Generate POT file             : `wp i18n make-pot wp-content/themes/astra wp-content/themes/astra/languages/astra.pot`
- [x] Add draft Release notes on github
- [x] Add draft changelogs to wpastra.com
- [x] Open PR to backport branch to `next-release`
---

### CODE REVIEW

File: inc/lib/astra-notices/class-astra-notices.php [hunk 1]
```diff
@@ -116,7 +116,7 @@ public function dismiss_notice() {
 			}
 
 			if ( false === wp_verify_nonce( $nonce, 'astra-notices' ) ) {
-				wp_send_json_error( esc_html_e( 'WordPress Nonce not validated.', 'astra' ) );
+				wp_send_json_error( esc_html_e( 'WordPress Nonce not validated.' ) );
```
Comment (nileshc-bsf) [inline inc/lib/astra-notices/class-astra-notices.php:119]:
```suggestion
				wp_send_json_error( esc_html_e( 'WordPress Nonce not validated.', 'astra' ) );
```

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 **Tags:** custom-menu, custom-logo, entertainment, featured-images, full-width-template, one-column, two-columns, left-sidebar, e-commerce, right-sidebar, custom-colors, editor-style, featured-images, full-width-template, microformats, post-formats, rtl-language-support, theme-options, threaded-comments, translation-ready, blog  
 **Tested up to:** 5.9
```
Comment (imnavanath) [inline README.md:6]:
```suggestion
**Tested up to:** 5.9.1  
```

File: readme.txt [hunk 3]
```diff
@@ -5,7 +5,7 @@ License URI: https://www.gnu.org/licenses/gpl-2.0.html
 Tags: custom-menu, custom-logo, entertainment, featured-images, full-width-template, one-column, two-columns, left-sidebar, e-commerce, right-sidebar, custom-colors, editor-style, featured-images, full-width-template, microformats, post-formats, rtl-language-support, theme-options, threaded-comments, translation-ready, blog
 Tested up to: 5.9
```
Comment (imnavanath) [inline readme.txt:6]:
```suggestion
Tested up to: 5.9.1
```

File: README.md (hunk 2 above)
Comment (nileshc-bsf) [reply README.md]: Done

File: readme.txt (hunk 3 above)
Comment (nileshc-bsf) [reply readme.txt]: Done

**Review by premanshup — APPROVED** [review]

File: style.css [hunk 4]
```diff
@@ -4,7 +4,7 @@ Theme URI: https://wpastra.com/
 Author: Brainstorm Force
 Author URI: https://wpastra.com/about/?utm_source=theme_preview&utm_medium=author_link&utm_campaign=astra_theme
 Description: Astra is fast, fully customizable & beautiful WordPress theme suitable for blog, personal portfolio, business website and WooCommerce storefront. It is very lightweight (less than 50KB on frontend) and offers unparalleled speed. Built with SEO in mind, Astra comes with Schema.org code integrated and is Native AMP ready so search engines will love your site. It offers special features and templates so it works perfectly with all page builders like Elementor, Beaver Builder, Visual Composer, SiteOrigin, Divi, etc. Some of the other features: # WooCommerce Ready # Responsive # RTL & Translation Ready # Extendible with premium addons # Regularly updated # Designed, Developed, Maintained & Supported by Brainstorm Force. Looking for a perfect base theme? Look no further. Astra is fast, fully customizable and WooCommerce ready theme that you can use for building any kind of website!
-Version: 3.7.7
+Version: 3.7.8
 Requires at least: 5.3
 Tested up to: 5.9
```
Comment (imnavanath) [inline style.css:9]:
```suggestion
Tested up to: 5.9.1
```

**Pull request merged**